mbed_greentea/mbed_target_info.py

```diff
@@ -381,82 +392,65 @@ def get_platform_property(platform, property):
     :return: property value, None if property not found
     """
 
-    # First load from targets.json if available
-    value_from_targets_json = get_platform_property_from_targets(platform, property)
-    if value_from_targets_json:
-        return value_from_targets_json
-
-    # Check if info is available for a specific platform
-    if platform in TARGET_INFO_MAPPING:
-        if property in TARGET_INFO_MAPPING[platform]['properties']:
-            return TARGET_INFO_MAPPING[platform]['properties'][property]
+    default = _get_platform_property_from_default(property)
+    from_targets_json = _get_platform_property_from_targets(
+        platform, property, default)
+    if from_targets_json or not default:
+        return from_targets_json
```

Comment:
Just wanna make sure I understand this logic. If target is not in targets.json, `from_targets_json` should always return `None`. If  for some reason `default` isn't set, then this should return `None` (the value of `from_targets_json`). That makes sense to me, I just want to make sure I understand this block since the logic wasn't immediately apparent to me when reading the code.

Reply:
That's correct, and the case 8 below.